webpack.config.js

```diff
@@ -165,18 +166,21 @@ if (!__TEST__) {
 // ------------------------------------
 // Rules
 // ------------------------------------
+const jsLoaders = [{
+  loader: 'babel-loader',
+  options: {
+    cacheDirectory: true,
+  },
+}]
+if (__DEV__) jsLoaders.unshift('react-hot-loader/webpack')
```

Comment:
Does this work as is?  I'm not as solid with webpack 2 yet, but the [docs](https://webpack.js.org/concepts/loaders/#configuration) show only this syntax:

```diff
-if (__DEV__) jsLoaders.unshift('react-hot-loader/webpack')
+if (__DEV__) jsLoaders.unshift({ loader: 'react-hot-loader/webpack' })
```